packages/dev/docs/src/PostListing.js

```diff
@@ -31,7 +31,7 @@ export function PostListing({type}) {
       {blogPages.map(page => <BlogPost key={page.name} {...page} />)}
       <div style={{display: 'flex', justifyContent: 'end'}}>
         <Link
-          href={`../${type}/${type}-feed.rss`}
+          href={`./${type}-feed.rss`}
```

Comment:
i think this is a problem with our website's configuration. i can't even visit https://react-spectrum.adobe.com/blog/blog-feed.rss, it keeps redirecting to the index.html appended url